js/pages/ssn.js

```diff
@@ -362,19 +556,101 @@ const displayModal = (onContinue) => {
     }
 }
 
-const digitMask = (event, rawInput) => {
-
+/**
+ * Updates the raw input string based on the user's interaction with an input field.
+ * This function adjusts the raw input by adding or removing characters based on the user's input
+ * and manages the positioning of characters considering any hyphens present in the value.
+ *
+ * @param {Event} event - The input event triggered by the user's interaction with the input field.
+ * @param {string} rawInput - The current raw input string before the event's changes.
+ * 
+ * @return {string} The updated raw input string after applying the changes from the input event.
+ */
+const setRawInput = (event, rawInput) => {
+    
     const input = event.data;
-    const position = event.target.selectionStart;
+    const value = event.target.value;
+    let position = event.target.selectionStart;
```

Comment:
Based on my reading, `selectionStart` is experimental in Chrome, and is currently not supported in other browsers.

Reply:
Will look into this.

Reply:
Can you reference what you were reading? Based on documentation[https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement] from Mozilla it appears that event is supported across multiple browsers:

![image](https://github.com/episphere/connectApp/assets/18222069/64201d5c-44af-45ab-bd12-697116c17334)

Reply:
On textareas and input forms selectionStart and selectionEnd are not experimental properties.  https://caniuse.com/?search=selectionStart

Reply:
We're good here.Thanks for the info.